brax/jumpy.py

```diff
@@ -30,17 +33,21 @@
 float32 = onp.float32
 int32 = onp.int32
 
+FORCE_JNP = os.getenv("JUMPY_FORCE_JNP", 'False').lower() in ('true', '1', 't')
```

Comment:
just to future-proof this (jumpy may handle also one day handle, say, torch tensors... who knows!), can we make this:

JUMPY_FORCE_LIB=JAX|NUMPY

and it forces jnp if jax, forces np if numpy?

Reply:
I think ideally that would be how this worked. Right now though, jumpy uses `onp` by default and only uses `jnp` when needed. What setting this flag to numpy should do in practice? 

Forcing `jnp` is needed because `vmap`, `scan`, etc cannot know what kinds of inputs the functions they return will be called on when jit is disabled. This use case is asymmetric, i.e., forcing the `jnp` branch in `_in_jit()` should always work (I think), but forcing the `onp` branch wouldn't work when the functions are called with jax inputs.

Forcing `jnp` cannot actually change what `_which_np()` (without arguments) returns, as explained in the comment below. Therefore, forcing `jnp` doesn't actually change jumpy to exclusively use `jnp`. So in this current form, this isn't really a flag for selecting which backed to use but rather a flag to circumvent the specific problem with `JAX_DISABLE_JIT`.

Reply:
OK, thanks for thinking through these edge cases.  I agree with the the points you laid out here.

Last suggestion, then: Given your line of thinking, the flag name "JUMPY_FORCE_JNP" might be too broad, because it's not really forcing jnp across the board.  So that leads me to two questions/suggestions:

1) What if we dropped the flag entirely, and instead `_in_jit()` always returns true if it detects `JAX_DISABLE_JIT` has been set?  This might be a bit too magical/unadvertised behavior... there might be a valid reason to set `JAX_DISABLE_JIT` and still use numpy code... but that's doubtful.  What do you think?

2) If that doesn't make sense, let's make the flag name a bit more specific: `JUMPY_IGNORE_DISABLE_JIT` or something.

Reply:
It's indeed hard to come up with a reasonable case where you would want to disable jit and use numpy. If this gets merged, it's almost certain someone will come up with one though 😄. Anyway, I think this might be adequate workaround for now. I believe the correct flag to check would then be `jax.config.jax_disable_jit`.